octobot_trading/enums.py

```diff
@@ -370,6 +409,7 @@ class ExchangeConstantsPositionColumns(enum.Enum):
     MAINTENANCE_MARGIN_RATE = "maintenance_margin_rate"
     STATUS = "status"
     SIDE = "side"
+    ORIGINAL_SIDE = "original_side"
```

Comment:
What will it be used for?

Reply:
its used in positions parser -> quantity_found to set a positive or negative quantity

Reply:
I don't understand why we need an `ORIGINAL_SIDE` when we already have `SIDE` for that.

Reply:
original side can be PositionSide.LONG or PositionSide.SHORT
while side can be PositionSide.BOTH, PositionSide.LONG or PositionSide.SHORT